docs/Settings.md

```diff
@@ -444,6 +444,7 @@
 | rx_spi_protocol |  |  |
 | rx_spi_rf_channel_count |  |  |
 | safehome_max_distance | 20000 | In order for a safehome to be used, it must be less than this distance (in cm) from the arming point. |
+| safehome_usage_mode | RTH | Used to control when safehomes will be used. |
```

Comment:
I suggest adding a brief explanation of the available modes here.

Reply:
Agreed.  Done